drivers/frequency/adf4368/adf4368.h

```diff
@@ -0,0 +1,661 @@
+#define ADF4368_REF_OK_MSK			NO_OS_BIT(3)
+#define ADF4368_ADC_BUSY_MSK			NO_OS_BIT(2)
+#define ADF4368_FSM_BUSY_MSK			NO_OS_BIT(1)
+#define ADF4368_LOCKED_MSK			NO_OS_BIT(0)
+
+/* ADF4368 REG005A Map */
+#define ADF4368_REG5A_RSV1			NO_OS_GENMASK(7, 2)
+#define ADF4368_VCO_CORE_MSK			NO_OS_GENMASK(1, 0)
+
+/* ADF4368 REG005B Map */
+#define ADF4368_CHIP_TEMP_LSB_MSK		NO_OS_GENMASK(7, 0)
+
+/* ADF4368 REG005C Map */
+#define ADF4368_REG5C_RSV1			NO_OS_GENMASK(7, 1)
+#define ADF4368_CHIP_TEMP_MSB_MSK		NO_OS_BIT(0)
+
+/* ADF4368 REG005E Map */
+#define ADF4368_VCO_BAND_MSK			NO_OS_GENMASK(7, 0)
+
+/* ADF4368 REG0060 Map */
+#define ADF4368_REG60_RSV1			NO_OS_GENMASK(7, 4)
+#define ADF4368_VCO_BIAS_MSK			NO_OS_GENMASK(3, 0)
+
+/* ADF4368 REG0063 Map */
+#define ADF4368_VERSION_MSK		        NO_OS_GENMASK(7, 0)
+
+
+
+#define ADF4368_SPI_4W_CFG(x)	(no_os_field_prep(ADF4368_SDO_ACTIVE_MSK, x) | \
+					no_os_field_prep(ADF4368_SDO_ACTIVE_R_MSK, x))
+
+#define ADF4368_BLEED_MSB_MSK			(ADF4368_COARSE_BLEED_MSK | \
+						 ADF4368_FINE_BLEED_MSB_MSK)
+
+#define ADF4368_SPI_SCRATCHPAD_TEST		0x5A
+
+/* Specifications */
+#define ADF4368_SPI_WRITE_CMD			0x0
+#define ADF4368_SPI_READ_CMD			0x8000
+#define ADF4368_SPI_DUMMY_DATA			0x00
+#define ADF4368_BUFF_SIZE_BYTES			3
+#define ADF4368_VCO_FREQ_MIN			6400000000U	// 6.4GHz
+#define ADF4368_VCO_FREQ_MAX			12800000000U	// 12.8GHz
+#define ADF4368_MOD1WORD			0x2000000U	// 2^25
+#define ADF4368_MOD2WORD_MAX			0xFFFFFFU	// 2^24 - 1
+#define ADF4368_PHASE_RESYNC_MOD2WORD_MAX	0x1FFFFU	// 2^17 - 1
+#define ADF4368_CHANNEL_SPACING_MAX		78125U
+#define ADF4368_PFD_FREQ_MAX			625000000U	// 625MHz
+#define ADF4368_PFD_FREQ_FRAC_MAX		250000000U	// 250MHz
+#define ADF4368_DCLK_DIV1_0_MAX			160000000U	// 160MHz
+#define ADF4368_DCLK_DIV1_1_MAX			320000000U	// 320MHz
+#define ADF4368_CLKOUT_DIV_REG_VAL_MAX		3
+
+#define ADF4368_RFOUT_MAX			12800000000U // 12.8GHz
+#define ADF4368_RFOUT_MIN			800000000U // 0.8GHz
+#define ADF4368_REF_CLK_MAX			4000000000U // 4GHz
+#define ADF4368_REF_CLK_MIN			10000000 // 10MHz
+#define ADF4368_REF_DIV_MAX			63
+#define ADF4368_OUT_PWR_MAX			15
+#define ADF4368_CPI_VAL_MAX			15
+#define ADF4368_BLEED_WORD_MAX			8191
+#define ADF4368_FRAC_N_INT_MIN			19
+#define ADF4368_INT_N_INT_MIN			4
+
+#define ADF4368_PHASE_BLEED_CNST		2044000
+#define ADF4368_BLEED_N_INT_TH			35
+#define ADF4368_COARSE_BLEED_CNST		202 // 202ua
+#define ADF4368_FINE_BLEED_CNST1		250 // 250ua
+#define ADF4368_FINE_BLEED_CNST2		512 // 250ua
+#define ADF4368_FINE_BLEED_CNST3		567 // 567na
+
+#define ADF4368_SIGMA_DELTA_MOD_CNST		4096 // 250ua
+
+#define ADF4368_POR_DELAY_US			200
+#define ADF4368_LKD_DELAY_MS			9
+
+#define MHZ					MEGA
+#define S_TO_NS					NANO
+#define NS_TO_PS				KHZ_PER_MHZ
+#define PS_TO_NS				KHZ_PER_MHZ
+#define PS_TO_US				MICRO
+#define US_TO_MS				MILLI
+#define MS_TO_S					MILLI
+#define PS_TO_S					PICO
+#define US_TO_FS				NANO
+#define NS_TO_S					NANO
```

Comment:
These names are generic enough to create future collision with other macros. If you really need to use them, please add the `ADF4368_` prefix.

Reply:
Thank you! Fixed